src/app/shared/directives/is-authenticated.directive.ts

```diff
@@ -0,0 +1,21 @@
+import { Directive, Input, OnInit, TemplateRef, ViewContainerRef } from '@angular/core';
+
+@Directive({
+  selector: '[appIsAuthenticated]'
+})
+export class IsAuthenticatedDirective implements OnInit{
+  @Input() appIsAuthenticated: boolean;
```

Comment:
I think, it will be better if this directive will not have this input argument. But instead you can inject AuthService directly in this directive

Reply:
i will implement it in the next task(when auth service will be finished)